src/main/java/com/dongtronic/diabot/commands/NightscoutCommand.kt

```diff
@@ -113,6 +115,39 @@ class NightscoutCommand(category: Command.Category) : DiabotCommand() {
         buildNightscoutResponse(endpoint, event)
     }
 
+    private fun processPebble(url: String, dto: NightscoutDTO, event: CommandEvent) {
+        val client = HttpClient()
+        val url_base = url.replace("/api/v1/", "/pebble")
+        val method = GetMethod(url_base)
+        val statusCode = client.executeMethod(method)
+
+        if (statusCode == -1) {
+            event.reactError()
```

Comment:
Consider logging the error using `logger.error()`

Reply:
Added.